src/cmd-buildextend-installer

```diff
@@ -148,15 +148,32 @@ def generate_iso():
                     tarinfo.mode = 0o0644
                 return tarinfo
 
-            # Install binaries from the grub2-efi and shim rpms (installed
-            # in our COSA container). Manually construct the tarball to ensure
-            # proper permissions and ownership
+            tmpimageefidir = tempfile.TemporaryDirectory()
```

Comment:
This should just be:
`tmpimageefidir = os.path.join(tmpdir, "efi")`
or so - we already allocate a tmpdir for the build, and it's better to have all temp files there.  That way we don't have separate tmpdirs that "leak" across builds potentially.

Reply:
Fixed in 8d66eed